src/coins.cpp

```diff
@@ -220,12 +234,29 @@ bool CCoinsViewCache::BatchWrite(CCoinsMap &mapCoins, const uint256 &hashBlockIn
 }
 
 bool CCoinsViewCache::Flush() {
-    bool fOk = base->BatchWrite(cacheCoins, hashBlock);
+    bool fOk = base->BatchWrite(cacheCoins, hashBlock, /*erase*/ true);
```

Comment:
You can add the = to these comments so that clang-tidy will check them.
```suggestion
    bool fOk = base->BatchWrite(cacheCoins, hashBlock, /*erase=*/ true);
```

Reply:
Fixed, thanks.